bucket/snaptimer.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "0.1",
+    "description": "Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://dan.hersam.com/software/license.php"
+    },
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": "sha1:7131F4C79038B8059F8EF16601077F075CFAD2DC",
+    "extract_dir": "SnapTimer",
+    "pre_install": [
+        "$FILE = 'SnapTimer.ini'",
+        "if (!(Test-Path \"$persist_dir\\$FILE\")) {",
+        "    Write-Host 'No' $FILE 'exists. Creating.' -f Yellow",
+        "    New-Item \"$dir\\$FILE\" -Type File | Out-Null",
+        "}"
+    ],
```

Do not use such code, when it can be inlined

```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\SnapTimer.ini\")) { New-Item \"$dir\\SnapTimer.ini\" | Out-Null }",
```